package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-template-imports",
-  "version": "2.0.1",
+  "version": "3.0.1",
```

Comment:
In general, bumping versions here should be done upon release. Can you revert this change?

Reply:
Updated. Thanks!